index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Portfolio-Mobile-First</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="wrapper">
+            <header>
+                <div class="logo">Welcome</div>
+                <button class="button-header">
+                    <img class="icone-clos" src="Images/Union.png" alt="-vector">
+                </button>
+            </header>
+            <main>
+                <section class="headline">
+                  <div class="content">
+                    <h1 class="title">
+                      Hey there. I'm <br >
+                      Jacinta 
+                    </h1>
+                    <h2 class="tiltle">
+                      I'm a software <br/>
+                      developer
+                    </h2>
+                    <p class="details">
+                      I can help you build a product , feature
+                      or website Look through some of my work and experience! If you like
+                      what you see and have a project you need coded, don’t hestiate to
+                      contact me.
+                    </p>
+                  </div>
+                  
+                  
+                  <ul class="social-media">
+                    <li>
+                      <a href="#"><img src="Images/twitter.png" alt="Twitter" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/Linkedin icon.png" alt="linkedin" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/medium.png" alt="medium" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/github.png" alt="github" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/angellist.png" alt="angellist" /></a>
+                    </li>
+                  </ul>
+                </section>
+        
```

Comment:
- [ ] This section has not been worked on as suggested by the previous reviewer. Kindly follow the guiding template you chose to work out your design